src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -207,6 +207,26 @@ entity corresponds to modality suffix,
 such as `T1w` or `inplaneT1`, referenced by the defacemask image.
 For example, `sub-01_mod-T1w_defacemask.nii.gz`.
 
+The OPTIONAL [`task-<label>`](../appendices/entities.md#task) entity can be used
+in order to allow tasks during structural MR acquisitions,
+for example pre-described motion paradigms such as nodding, to be described.
+Task labels MUST be consistent across subjects and sessions.
+
+### RECOMMENDED metadata for tasks in anatomical imaging
+
+<!-- This block generates a metadata table.
+The definitions of these fields can be found in
+  src/schema/objects/metadata.yaml
+and a guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_metadata_table(
+   {
+      "TaskName": "RECOMMENDED",
+      "TaskDescription": "RECOMMENDED",
+      "Instructions": "RECOMMENDED",
+   }
+) }}
```

Comment:
```suggestion
) }}

```

pretty sure we need to skip a line for the table generation macro to not start using the next paragraph as input

@tsalo seen that happen before?